utils/__tests__/config.test.ts

```diff
@@ -0,0 +1,199 @@
+import { Either } from "fp-ts/lib/Either";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import { MailerConfig } from "../config";
+
+const aMailFrom = "example@test.com";
+
+const noop = () => {};
+const expectRight = <L, R>(e: Either<L, R>, t: (r: R) => void = noop) =>
+  e.fold(
+    _ =>
+      fail(`Expecting right, received left. Value: ${JSON.stringify(e.value)}`),
+    _ => t(_)
+  );
+
+const expectLeft = <L, R>(e: Either<L, R>, t: (l: L) => void = noop) =>
+  e.fold(
+    _ => t(_),
+    _ =>
+      fail(`Expecting left, received right. Value: ${JSON.stringify(e.value)}`)
+  );
```

Comment:
Nice approach. Can we move these methods to a test utility library to share this logic across all the projects?

Reply:
I'd like to write a jest extension module with custom matchers for io-ts/fp-ts. Maybe one day ;)
There are still quirks with the above functions (failures aren't reported very well)